packages/terminology/src/router.ts

```diff
@@ -0,0 +1,67 @@
+import { Router, Request, Response } from "express";
+import { codeSystemImportHandler } from "./operations/codeImport";
+import { codeSystemLookupHandler } from "./operations/codeLookup";
+import { conceptMapImportHandler } from "./operations/conceptMapImport";
+import { conceptMapTranslateHandler } from "./operations/conceptMapTranslate";
+import { FhirRequest, HttpMethod } from "@medplum/fhir-router";
+import { asyncHandler } from "./util";
+
+const fhirRouter = Router();
+
+function createFhirRequest(req: Request): FhirRequest {
+  return {
+    method: req.method as HttpMethod,
+    pathname: req.originalUrl.replace("/fhir/R4", "").split("?").shift() || "",
+    params: req.params,
+    query: req.query as Record<string, string>,
+    body: req.body,
+    headers: req.headers,
+  };
+}
+
+fhirRouter.post(
+  "/CodeSystem/import",
+  asyncHandler(async (req: Request, res: Response) => {
+    const fhirRequest = createFhirRequest(req);
+    const response = await codeSystemImportHandler(fhirRequest);
+    res.status(200).json(response);
+  })
+);
+
+fhirRouter.post(
+  "/CodeSystem/lookup",
+  asyncHandler(async (req: Request, res: Response) => {
+    const fhirRequest = createFhirRequest(req);
+    const response = await codeSystemLookupHandler(fhirRequest, false);
+    res.status(200).json(response);
```

Comment:
We're returning arrays as the response of these endpoints. Any particular reason for this?

We standardized our responses to be a JSON object, because its easier to extend than an array.